public/js/eventsPage.js

```diff
@@ -228,28 +228,32 @@ async function eventGenerator(event, hiringPartners) {
         eventInformation += `<p>Notes: ${event.notes}</p>`
     }
 
-    eventInformation += `<div class="event-attendees">`
-    eventInformation += `<div class="hiring-partners col-xs-12 col-md-6" data-eventId='${event.id}'></div>`;
-    if (!isPastPage) {
-        eventInformation += `
-            <div class='addHiringPartner col-xs-12 col-md-6'>
-                <h5>Add attendees</h5>
-                <form class='addHiringPartnerForm' id='${event.id}'>
+    if (event.availableToHP == 1) {
```

Comment:
dont need the comparison, 1 is true anyway, so just do `if (event.availableToHP)`